### PR TITLE
Handle errors & add more payloads

### LINKED ISSUE #9: value: Timeout
Can you help me to fix this error 
Thanks 

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Timeout', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/ppfuzz-0.0.1/src/browser.rs:23:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Capture

### PR DESCRIPTION
**Pull request by dwisiswant0**
## Proposed of changes

- Update installation usage
- Handle invalid data from file
- Add pointer payloads
- Update default concurrency level (15)
- Handle error on launch browser (fix #9)

### CODE REVIEW

File: src/builder/payload.rs [hunk 1]
```diff
@@ -1,13 +1,37 @@
...
 pub fn get() -> Vec<String> {
+	let mut payload = vec![];
+
+	let object = get_object();
+	let pointer = get_pointer();
+
+	payload.extend(object);
+	payload.extend(pointer);
+
+	payload
```
Comment (mattsse) [inline src/builder/payload.rs:10]:
```suggestion
	vec![ get_object(), get_pointer() ]
```

File: src/builder/payload.rs [hunk 2]
```diff
@@ -1,13 +1,37 @@
...
+	const PREFIX: &str = "__proto__";
 	let suffixes = [
 		".ppfuzz",
 		"[ppfuzz]"
 	];
 
 	suffixes
 		.iter()
-		.map(|suffix| PROTO.to_owned() + suffix)
+		.map(|suffix| PREFIX.to_owned() + suffix)
```
Comment (mattsse) [inline src/builder/payload.rs:22]:
```suggestion
		.map(|suffix|  format!("{}{}", PREFIX, suffix))
```

File: src/builder/payload.rs [hunk 3]
```diff
@@ -1,13 +1,37 @@
...
+fn get_pointer() -> Vec<String> {
+	const PREFIX: &str = "constructor";
+	let suffixes = [
+		".prototype.ppfuzz",
+		"[prototype][ppfuzz]"
+	];
+
+	suffixes
+		.iter()
+		.map(|suffix| PREFIX.to_owned() + suffix)
```
Comment (mattsse) [inline src/builder/payload.rs:35]:
```suggestion
		.map(|suffix| format!("{}{}", PREFIX, suffix))
```

**Review by mattsse — APPROVED** [review]
minor nits

File: src/builder/payload.rs (hunk 1 above)
Comment (dwisiswant0) [reply src/builder/payload.rs:10]: It says
```
expected struct `std::string::String`, found struct `Vec`
```

Comment (mattsse) [reply src/builder/payload.rs:10]: Ah my bad, missed that get_* return vecs as well, then that's fine, only suggestion would be:

```rust
let mut payload = get_object();
payload.extend(get_pointer());
```

Comment (dwisiswant0) [reply src/builder/payload.rs]: @mattsse, I ended up with `concat`. :smile:

Comment (mattsse) [reply src/builder/payload.rs]: yeah, there way too many ways to write this and all probably end up the same

Comment (dwisiswant0) [reply src/builder/payload.rs]: Can you please review this one more?

**Review by mattsse — APPROVED** [review]

**Pull request merged**